### DISCUSSION TITLE
Add active highlighting for searching

### DISCUSSION THREAD

**Discussion by ALazyMeme** [Chatterino/chatterino2#3476] [Ideas]
Active highlighting meaning it will highlight the part of the message it matches with what was searched. No idea how it'd work for emotes though, but it'd be nice if it were there for text.

Rough example below:

![image](https://user-images.githubusercontent.com/12804673/148647719-85d88a53-4170-4f12-8b41-a6708a65bc0e.png)

**Comment by pajlada** [comment]
If we highlight the background instead, we could apply this to emotes. Maybe with some "halo" effect (i.e. it'd highlight the emote size + some more), it could be clear also for full-size emotes.

**Reply by leon-richardt** [reply]
:angel:

**Comment by Mm2PL** [comment]
Accepted as issue #5220.
